.travis.yml

```diff
@@ -28,6 +28,11 @@ jobs:
         # submit coverage.txt to codecov.io
         - bash <(curl -s https://codecov.io/bash)
 
+    - &mac-test
```

Comment:
What is this test for?

Reply:
> What is this test for?

YAML alias for sharing the settings across unit test on macOS

Reply:
ok